db-integrity-tests/src/checker.py

```diff
@@ -0,0 +1,109 @@
+"""Class to check attributes read from the AWS S3."""
+
+import datetime
+import re
+
+
+class Checker:
```

Comment:
I think we discussed this before, but is this the same as inheriting from Object, in Python 3?

Reply:
IMHO in Python 3 it is the same as Checker(object). But this is very good point - I'll update the code to explicitly use new-style classes syntax, because in Python 2 the behaviour is a bit different (old-style vs. new-style). Thanks :point_up_2:

Reply:
Yes, in [Python 3 it's the same](https://stackoverflow.com/questions/15374857/should-all-python-classes-extend-object#answer-15374884)